phase2/Kconfig

```diff
@@ -39,6 +39,14 @@ config phase2.kubeadm.version
 
 	  Valid options are "stable" or a Google Cloud Storage link to a build.
 
+config phase2.kubeadm.master_upgrade.method
+	string "kubeadm master upgrade method"
+	default "upgrade"
+	help
+	  The kubeadm method of master upgrade to use.
+
+	  Valid options are upgrade.
```

Comment:
I would definitely list all valid options, instead of hiding any. If we do have different levels of support or recommendations for the options to use, we can document those in the help text, but in a project whose primary goals are to make everything easy to read and understand, it seems wrong to deliberately omit options.